content/en/service_management/incident_management/guides/_index.md

```diff
@@ -20,6 +20,11 @@ cascade:
 {{< whatsnext desc="Integrations:">}}
     {{< nextlink href="/service_management/incident_management/guides/jira" >}}Jira{{< /nextlink >}}
     {{< nextlink href="/service_management/incident_management/guides/statuspage" >}}Statuspage{{< /nextlink >}}
+    {{< nextlink href="/integrations/servicenow/#configure-incident-management" >}}ServiceNow{{< /nextlink >}}
+    {{< nextlink href="/integrations/zoom_incident_management" >}}Zoom{{< /nextlink >}}
+    {{< nextlink href="/coscreen/?tab=desktop#coscreen--datadog-incident-management" >}}CoScreen{{< /nextlink >}}
+    {{< nextlink href="/integrations/statuspage" >}}Statuspage Integration{{< /nextlink >}}
```

Comment:
```suggestion
    {{< nextlink href="/integrations/statuspage" >}}Statuspage (integration){{< /nextlink >}}
```

Reply:
The difference between the two statuspage links here was confusing for a moment, so I'm suggesting this update to clarify

Reply:
Ah, I feel silly, I was on autopilot. I'll delete this link. I have it linked in the other guide.